testing_openstack_code_changes_with_operators.md

```diff
@@ -124,3 +124,47 @@ Status:
       /var/lib/kolla/config_files/config.json from config (ro,path="neutron-api-config.json")
       /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-rhk5m (ro)
 ```
+
+# Testing OpenStack code changes in the CI
+
+With [recent change](https://github.com/openstack-k8s-operators/ci-framework/pull/1892)
+in the ci-framework it is now possible to run operator's CI jobs together with
+the depends-on patch proposed for the upstream OpenStack project in upstream
+gerrit.
+
+This require to replace `openstack-k8s-operators-content-provider` job with
```

Comment:
LGTM

@raukadah wondering why we didn't switched openstack-k8s-operators-content-provider itself to support this so the capability to test upstream changes auto enabled on all repos? just checking if there is some benefit in maintaining both these jobs going forward.

Reply:
I think that @raukadah's idea behind this is to keep it like that until GA and then we can change that later